src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -0,0 +1,49 @@
+package uk.gov.nationalarchives.tdr.api.graphql.fields
+
+import java.util.UUID
+
+import sangria.marshalling.circe._
+import io.circe.generic.auto._
+import sangria.ast.StringValue
+import sangria.schema.ScalarType._
+import sangria.macros.derive._
+import sangria.schema.{Argument, Field, InputObjectType, ListType, ObjectType, ScalarType, fields}
+import sangria.validation.ValueCoercionViolation
+import uk.gov.nationalarchives.tdr.api.graphql.ConsignmentApiContext
+
+import scala.util.{Failure, Success, Try}
+
+object ConsignmentFields {
+  private case object UuidCoercionViolation extends ValueCoercionViolation("Valid UUID expected")
```

Comment:
May be split this out, as it is likely UUID are going to be used elsewhere for other objects